### ISSUE TITLE
[Guestbook] ACP config doesn't make sense

### ISSUE THREAD

**Issue by vendethiel** [MightyGorgon/icy_phoenix_plugins#26]
Same as for blogs, what am I supposed to see in the ACP? or is it just a dummy file?

How am I supposed to add guestbooks? not sure

**severity: major** (or at least I think so...)

**Comment by MWE001** [comment]
Not sure this, like the blogs, even needs ACP interface in my opinion. The only real option needed in the ACP would be to turn off or on guest posting and override user settings. If the user settings to allow ALL to reply is taken away, no ACP interface is going to be needed at all. I don't think. that wold save MAJOR time with not having to code a back end. Simply remove a sql query and a line or 3 of code in guestbook.php or what ever file it is that allows ALL to reply and bingo bango! Mission accomplished. Do away with ACP options all together.

**Comment by vendethiel** [comment]
@MWE001 maybe. It's a bit surprising to not have an acp page for some plugin appear; though :).

**Comment by MWE001** [comment]
yeah I was a little surprised when I installed that saduku or what ever it is once just to see what it is and it had no acp.... that I saw anywhere. 

If I had a vote in the matter, I think the only ACP options it would need is to set defaults that will over ride user preferences as to who can post and who can reply and view etc etc. Just like the blogs needs. You could dang near get away with writing one back end and adapting it for both. But I'm not a pro coder so I really don't know  lol just a guess from left field.
